z3/src/sort.rs

```diff
@@ -212,6 +212,8 @@ impl<'ctx> Sort<'ctx> {
             Z3_mk_int64(self.ctx.z3_ctx, i, self.z3_sort)
         })
     }
+*/
+
```

Comment:
I commented out the `Sort::from_u64(&self, u: u64)` and `Sort::from_i64(&self, i: i64)` methods; if you're okay with that, we can just remove them before merging rather than leaving them commented out

Reply:
Okay to remove, I think.